dev-libs/xapian/xapian-1.2.25.recipe

```diff
@@ -48,21 +47,29 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libuuid$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libuuid$secondaryArchSuffix >= 1.3
 	lib:libz$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
 	xapian${secondaryArchSuffix}_devel = $portVersion
 	devel:libxapian$secondaryArchSuffix = $libVersionCompat
+	cmd:xapian_config$secondaryArchSuffix = $portVersion
 	"
 REQUIRES_devel="
 	xapian$secondaryArchSuffix == $portVersion base
 	"
+if [ -n "$secondaryArchSuffix" -a "$commandBinDir" != "$prefix"/bin ]; then
```

Comment:
```diff
-if [ -n "$secondaryArchSuffix" -a "$commandBinDir" != "$prefix"/bin ]; then
+if [ -n "$secondaryArchSuffix" -a "$commandBinDir" == "$binDir" ]; then
```

The less hardcoding, the better :)

Reply:
Yeah, I do agree that `"$prefix"/bin` is a hardcoded path, but since we already have it earlier, using `[ -n "$secondaryArchSuffix" -a "$commandBinDir" != "$prefix"/bin ]` instead of `[ -n "$secondaryArchSuffix" -a "$commandBinDir" = "$binDir" ]` makes the test easier to translate to english.
The sentence I would translate this to is:
_Are we building for a secondary arch and are we installing on something other than `"$prefix"/bin`?_

After all, we want a conflicts _only_ if these two conditions are both met:
* we are building for a secondary arch
* we did not change the default definition for `commandBinDir`.

Reply:
After all, since I had to fix a typo, I switched the test to use your variant :)